pennylane/ops/op_math/condition.py

```diff
@@ -48,12 +46,7 @@ class Conditional(Operation):
 
     num_wires = AnyWires
 
-    def __init__(
-        self,
-        expr: MeasurementValue[bool],

```

Comment:
did importing this now cause a circular dependency? might want to use strings to keep the type-hints

Reply:
Good point, will do